modules/core/test/test_intrin_utils.hpp

```diff
@@ -1736,49 +1736,59 @@ template<typename R> struct TheTest
     }
 #endif
 
-#if CV_SIMD_64F
-    TheTest & test_cmp64()
+    void do_check_cmp64(const Data<R>& dataA, const Data<R>& dataB)
     {
-        Data<R> dataA, dataB;
-        R a = dataA, b = dataB;
-
-        for (int i = 0; i < VTraits<R>::vlanes(); ++i)
-        {
-            dataA[i] = dataB[i];
-        }
-        dataA[0]++;
-
-        a = dataA, b = dataB;
+        R a = dataA;
+        R b = dataB;
 
-        Data<R> resC = (a == b);
-        Data<R> resD = (a != b);
+        Data<R> dataEQ = (a == b);
+        Data<R> dataNE = (a != b);
```

Comment:
This doesn't work with RVV "scalable" mode "as is".

Fix for this file should look like this (patch for 4.x was in history of #23307 which GitHub don't want to show anymore):

```.diff
@@ -1741,8 +1741,13 @@ template<typename R> struct TheTest
         R a = dataA;
         R b = dataB;
 
+#if CV_SIMD_SCALABLE
+        Data<R> dataEQ = v_eq(a, b);
+        Data<R> dataNE = v_ne(a, b);
+#else
         Data<R> dataEQ = (a == b);
         Data<R> dataNE = (a != b);
+#endif
 
         for (int i = 0; i < VTraits<R>::vlanes(); ++i)
         {
@@ -1764,6 +1769,7 @@ template<typename R> struct TheTest
     {
         Data<R> dataA;
         Data<R> dataB;
+
         for (int i = 0; i < VTraits<R>::vlanes(); ++i)
         {
             dataA[i] = dataB[i];
@@ -2136,7 +2142,8 @@ void test_hal_intrin_float64()
         .test_rotate<2>().test_rotate<3>()
 #endif
         ;
-
+#else
+    std::cout << "SKIP: CV_SIMD_64F is not available" << std::endl;
 #endif
 }
 
@@ -2156,6 +2163,7 @@ void test_hal_intrin_float16()
 #endif
 }
 
+
 /*#if defined(CV_CPU_DISPATCH_MODE_FP16) && CV_CPU_DISPATCH_MODE == FP16
 void test_hal_intrin_float16()
 {
```